docs: rabbit-mq release notes v3.10.7 (#412)
elastisys/compliantkubernetes: docs/release-notes/rabbitmq.md

!!!note
    These are only the user-facing changes.

### v3.10.7-ck8s1

### Release notes

> 

Reviewer: ```suggestion

Released 2022-09-21

Changes:

```

Author: Thanks @davidumea , This has been fixed now !!

> - Changed versioning to be based on the server version rather than the operator version. The version is now <server version>-<ck8s patch>, e.g. v3.10.7-ck8s1. Read more about versioning and upgrades at `docs/upgrades-and-patches.md`.

Reviewer: ```suggestion
- Changed versioning to be based on the server version rather than the operator version. The version is now <server version>-<ck8s patch>, e.g. v3.10.7-ck8s1.
```

Author: Thanks @davidumea , This has been fixed now !!

> ### Fixed
> 
> - Fixed anti affinity rules to match the labels

Reviewer: ```suggestion
```

Author: Thanks @davidumea , This has been fixed now !!

> ### Added
> 
> - Added alert for definitions backup job
> - Added monitor target for detailed metrics from the prometheus plugin
> - Added queue details Grafana dashboard based on metrics from the prometheus plugin, made to replace the metrics exporter dashboard
> - Added affinity and tolerations to the cluster operator deployment
> - Added support for external access using load balancer or node port service
> - Added issuer to generate chain of trust for TLS support

Reviewer: ```suggestion

- Improved monitoring and alerting
- Added queue details Grafana dashboard based on metrics from the prometheus plugin, made to replace the metrics exporter dashboard
- Added support for external access using load balancer or node port service
- Added issuer to generate chain of trust for TLS support
```

Author: Thanks @davidumea , This has been fixed now !!

> ### Updated
> 
> - Updated cluster operator to v1.14.0, updated server to 3.10.7
> - Updated patching docs to include when to upgrade

Reviewer: ```suggestion

- Updated cluster operator to v1.14.0, updated server to 3.10.7
```

Author: Thanks @davidumea , This has been fixed now !!

> ### Changed
> 
> - Set default server version in cluster chart to v3.10.7
> - Updated some of our rules to better handle non-standard/multi-cluster setups
> - Updated scripts and snippets to yq4

Reviewer: ```suggestion
- Improved support for multi-cluster setup
```

Author: Thanks @davidumea , This has been fixed now !!

> - Split RBAC for admins and devs

Reviewer: Do we need to write something on the public docs page about the split rbac?

https://elastisys.io/compliantkubernetes/user-guide/additional-services/rabbitmq/

Author: Thanks @davidumea i will review from `users-perspective` relevant informations changes they needed and will update accordingly !!

Author: Thanks @davidumea , This has been fixed now !!

> - Track server version instead of cluster operator version
> 
> ### Removed
> 
> - Removed alert for unroutable messages
> - Removed metrics exporter and dashboard
> 

Reviewer: ```suggestion

- Removed metrics exporter and dashboard

```

Author: Thanks @davidumea , This has been fixed now !!
